gpytorch/likelihoods/gaussian_likelihood.py

```diff
@@ -22,7 +22,7 @@ def forward(self, input):
         assert isinstance(input, GaussianRandomVariable)
         mean, covar = input.representation()
         noise = add_diag(covar, self.log_noise.exp())
-        return GaussianRandomVariable(mean, noise)
+        return input.__class__(mean, noise)
```

Comment:
this seems unnecessary general, given that we explicitly check that `input` is an GaussianRandomVariable above

Reply:
Right now, `MultitaskGaussianRandomVariable` is a `GaussianRandomVariable`. I was doing this to preserve that (this way, `GaussianLikelihood` can still be used with multitask if you don't need different task variances